README.md

```diff
@@ -2,9 +2,15 @@
 
 ## Installation
 
-To build locally run:
+This project is not currently published to any public repositories. You will need to install the JARs manually.
 
-    gradle build shadowjar
+The latest builds can be found here: https://github.com/stripe/stripe-java/releases/latest
```

Comment:
Need to update this to our repo then we should be good.

Reply:
🤦‍♂️